app/src/main/java/chat/rocket/android/password/ui/PasswordFragment.kt

```diff
@@ -17,6 +18,7 @@ import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.disposables.Disposable
 import io.reactivex.rxkotlin.Observables
 import kotlinx.android.synthetic.main.fragment_password.*
+import kotlinx.android.synthetic.main.app_bar.*
 import javax.inject.Inject
 
 class PasswordFragment: Fragment(), PasswordView, ActionMode.Callback {
```

Comment:
Hi @divyanshub024, can you move the `companion object` to the end of the class, so that we can follow more closely the Kotlin [guidelines](https://kotlinlang.org/docs/reference/coding-conventions.html#class-layout).